artifactory/commands/golang/archive.go

```diff
@@ -152,7 +147,36 @@ func archiveProject(writer io.Writer, dir, mod, version string) error {
 		return err
 	}
 
-	return Create(writer, m, files)
+	return gozip.Create(writer, m, files)
+}
+
+func isPathExcluded(path string, excludePathPattern string) (excludedPath bool, err error) {
+	if len(excludePathPattern) > 0 {
+		excludedPath, err = regexp.MatchString(path, excludePathPattern)
+	}
+	return
+}
+
+func prepareExcludePathPattern(exclusions []string, patternType utils.PatternType) (string, error) {
```

Comment:
This function is pretty much a duplicate of the below function in jfrog-client-go.
We should avoid code duplications.
<img width="1112" alt="image" src="https://github.com/jfrog/jfrog-cli-core/assets/7830056/6571da36-64f8-4585-bfa0-0c8008ce9532">

Reply:
Done